### ISSUE TITLE
vSphere CPI and CSI configuration is not visible on v2.6-head rancher.

### ISSUE THREAD

**Issue by vivek-infracloud** [rancher/dashboard#5192]
**Rancher Server Setup**
- Rancher version: v2.6-head (11a7451)
- Installation option (Docker install/Helm Chart): Docker

**Information about the Cluster**
- Kubernetes version:
- Cluster Type (Local/Downstream):
   - If downstream, what type of cluster? Downstream RKE2.

   
**Describe the bug**

vSphere CPI and CSI configuration is not visible on v2.6-head rancher.

**To Reproduce**
 

1. Install v2.6-head rancher.
2. Proceed to create RKE2 vSphere cluster.
3. Input correct vSphere credentials.
4. Input the required pool, instance options.
5. In Cluster Configuration, select Cloud Provider as vSphere. You will be prompted to configure vSphere CPI and CSI configuration in Add-On Config tab.

> `Important: Configure the vSphere Cloud Provider and Storage Provider options in the Add-On Config tab.
`

6. Click on the Add-On Config.

**Result**
A form to enter the vSphere CPI Configuration and vSphere CSI Configuration is not visible.

![Screenshot from 2022-02-18 15-30-30](https://user-images.githubusercontent.com/83208989/154666876-7d299ee3-1c28-40a8-a5a6-046799cf9700.png)

**Expected Result**

A form to enter the vSphere CPI Configuration and vSphere CSI Configuration should be visible just below the Calico Configuration.

![Screenshot from 2022-02-18 15-30-25](https://user-images.githubusercontent.com/83208989/154669723-ee2d373a-9edb-43e9-bc02-d6a8e5eede72.png)

**Comment by paynejacob** [comment]
Given the issue is just the form appearing and not appearing we believe this is a dashboard issue.

**Comment by codyrancher** [comment]
I just pulled the latest of v2.6-head and I'm not seeing this issue. Would you mind checking to see if you can still reproduce this?

![image](https://user-images.githubusercontent.com/55104481/156448994-240cbd5d-424f-4906-82ea-618ad599b50a.png)

**Comment by vivek-infracloud** [comment]
**Tested this issue on vSphere Environment 7 on latest v2.6-head(39befda) rancher.** **Issue is not active on latest rancher v2.6-head.** 

![Screenshot from 2022-03-03 12-54-32](https://user-images.githubusercontent.com/83208989/156516957-1b1d75c5-6a21-4cae-9ed9-164214e7636e.png)


**However, the issue still active on v2.6.4-rc4.**
![Screenshot from 2022-03-03 12-31-59](https://user-images.githubusercontent.com/83208989/156517041-bd486cf6-353b-4485-8aae-00ef7499b0e6.png)

@codyrancher Not sure if I should close the issue.

**Comment by vivek-infracloud** [comment]
Since the v2.6-head is ahead of the v2.6.4-rc4. I'm considering the issue as fixed. Hence, closing the issue.

**Comment by anupama2501** [comment]
Re-opening for another validation on the latest rc.

**Comment by vivek-infracloud** [comment]
Blocked by this issue - https://github.com/rancher/dashboard/issues/5462

**Comment by vivek-infracloud** [comment]
Issue is active on rancher v2.6.4-rc11.

**Comment by mantis-toboggan-md** [comment]
This is a backend issue and duplicate of https://github.com/rancher/rancher/issues/36987
